packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -167,10 +168,12 @@ const taskDefinition = new ecs.TaskDefinition(this, 'TaskDef', {
 Images supply the software that runs inside the container. Images can be
 obtained from either DockerHub or from ECR repositories:
 
-* `ecs.DockerHub.image(imageName)`: use a publicly available image from
+* `ecs.ContainerImage.fromDockerHub(imageName)`: use a publicly available image from
   DockerHub.
-* `repository.getImage(tag)`: use the given ECR repository as the image
+* `ecs.ContaienrImage.fromEcrRepository(repo, tag)`: use the given ECR repository as the image
   to start.
+* `ecs.ContainerImage.fromAsset({ directory: './image' })`: build and upload an
```

Comment:
I'd make `directory` the first positional argument and options as an optional, so usage will be:

```ts
ecs.ContainerImage.fromAsset('./image');
ecs.ContainerImage.fromAsset('./image', { options... });
```